airbyte-cdk/python/unit_tests/sources/declarative/test_concurrent_declarative_source.py

```diff
@@ -95,7 +136,8 @@
             "error_message": "No data available for the time range requested."
           }
         ]
-      }
+      },
+      "use_cache": False,  # necessary to ensure all tests are independent and that HttpMocker can validate the HTTP requests
```

Comment:
I still couldn't get the HttpMocker validation that all HTTP requests were performed because of substream caching so I'll remove this